packages/medusa-cli/src/create-cli.js

```diff
@@ -54,12 +56,17 @@ function buildLocalCommands(cli, isLocalProject) {
       )
       return require(cmdPath).default
     } catch (err) {
+      if (process.env.NODE_ENV !== "production") {
```

Comment:
I like it

Reply:
Question: all the places where we explicitly use production, test and such words for the node env. Should we put them somewhere and make them configurable? The thinking here is that someone could easily set the NODE_ENV to prod and we won’t see it. I might overthinking it, and we might also not care too much about that but it is just a thought

Reply:
util functions to identify that would be good, like `const isProduction = ["production", "prod"].includes(process.env.NODE_ENV || "")`.
But I'm not about why make that configurable or why. This could respect some sort of "LogLevel" though.

Reply:
I am fine with the util, it is just if someone set it with something different but I don’t think it would happen. I hope there is a minimal consensus around the world 😂

Reply:
We might also document that in the documentation wdyt? @olivermrbl @shahednasser

Reply:
ok, now I see what you meant by configurable.
Having a list of these words somewhere in the config might be nice.

Reply:
I like the idea as well, but I am fine keeping it for a separate PR :)